website/docs/r/aviatrix_transit_gateway_peering.html.markdown

```diff
@@ -49,6 +49,9 @@ The following arguments are supported:
 * `prepend_as_path1` - (Optional) AS Path Prepend customized by specifying AS PATH for a BGP connection. Applies on transit_gateway_name1. Available in provider version R2.17.2+.
 * `prepend_as_path2` - (Optional) AS Path Prepend customized by specifying AS PATH for a BGP connection. Applies on transit_gateway_name2. Available in provider version R2.17.2+.
 * `enable_peering_over_private_network` - (Optional) Enable peering over private network. ActiveMesh and Insane Mode is required on both transit gateways. Available in provider version R2.17.1+.
+* `enable_single_tunnel_mode` - (Optional) Enable peering with Single Tunnel mode. False by default.
+  
+~> **NOTE:** Basically the peering is **not** in "Single Tunnel" mode only when `enable_peering_over_private_network` is `true` and `enable_single_tunnel_mode` is `false`. When `enable_peering_over_private_network` is `false`, the "Single Tunnel" peering is public.
```

Comment:
Don't need the word 'Basically', lets keep the docs more formal.

Also this note is a bit confusing to me, why do we say that "the peering is **not** in "Single Tunnel" mode only when `enable_peering_over_private_network` is `true` and `enable_single_tunnel_mode` is `false`."? Isn't the tunnel **not** in single tunnel mode whenever `enable_single_tunnel_mode` is set to False?

Reply:
Even the backend thinks this new "single_tunnel" feature probably will be confusing. For instance, if you peer two normal gateways (not insane mode), they are in fact connected by a "single tunnel". 

This new feature is for those who just want to peer two insane gateways over private network by a single tunnel. Maybe using the release note here is better.

"Private Transit Gateway Peering with Single-Tunnel Mode expands the existing Insane Mode Transit Gateway Peering Over Private Network to apply it to single IPSec tunnel. One use case is for low speed encryption between cloud networks."

Reply:
Got it, in this case, we should block the use of this attribute if `enable_peering_over_private_network` is not true. An error message like "`enable_single_tunnel_mode` is only valid when `enable_peering_over_private_network` is set to true" should help the users understand what is going on.

We should then have the docs say that this attribute is only valid with  `enable_peering_over_private_network`.

Also please add "Available as of provider version R2.18+."